### DISCUSSION TITLE
For those with meters in North America without regular consumption LEDs

### DISCUSSION THREAD

**Discussion by bean72** [klaasnicolaas/home-assistant-glow#304] [General]
First of all, I want to thank @klaasnicolaas for putting this together, I've jumped so many hoops to try and get my meter readings when I should have gone straight to this.
Some backstory, I've got a Centron meter that I have been trying to pull data from for a number of months now. It's Zigbee energy profile capable, but any hubs that connect to the meter have been out of stock or crazy prices. I tried pulling the SCM using an SDR without luck, as it is an Openway meter and everything is encrypted. I was able to find the Technical Reference Guide for my meter, and discovered that there is an IR LED in the very top that emits based on usage.
![image](https://github.com/klaasnicolaas/home-assistant-glow/assets/37682005/597bdfe8-4123-441e-aac7-78f71e2e3f21)

![image](https://github.com/klaasnicolaas/home-assistant-glow/assets/37682005/e8f8b5f8-fefd-4ab9-a80d-8b17efb0b8a6)

Rather than using a photodiode, I wired an ESP to an IR receiver diode using the Home Assistant Glow code. So far its working great except when the sun is shining directly on my meter, which I'll have to craft something to block the light.

If you are in North America with a similar meter, this should work for you. Thanks again!

**Comment by rspilk** [comment]
My concern is the utility thinking you are doing something nefarious to impact metering of energy. Don't they get weird about attaching things to the meter thinking they might be magnets or something?
